### PR TITLE
RFC-2119 github workflow

### LINKED ISSUE #1902: add github action to prevent errant RFC-2119 terms sneaking back in
# Describe the change

We should prevent lower case RFC-2119 normative words. Any uses of RFC-2119 MUST be uppercase in the ARIA spec.

# Link to the version of the specification or documentation you were looking at

Link to documentation:

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)?

### PR DESCRIPTION
**Pull request by ariellalgilmore**
Closes #1902 

worked with @andreancardona!

Adds a new github workflow for new PR's that checks if the PR is introducing any lower case versions of RFC-2119 terminology

### CODE REVIEW

**Comment by pkra** [comment]
This looks really helpful! Thank you @ariellalgilmore & @andreancardona!

@daniel-montalvo is there a W3C policy on third party github actions?

**Review by daniel-montalvo — APPROVED** [review]

File: .github/workflows/RFC-2119-usage.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check PR
+      uses: JJ/github-pr-contains-action@releases/v10
+      with:
+        github-token: ${{github.token}}
+        diffDoesNotContain: " must | must not | required | shall | shall not | should | should not | recommended | may | optional "
```
Comment (spectranaut) [inline .github/workflows/RFC-2119-usage.yml:16]: remove "required", "optional", "recommended" and make follow up issue to discuss removing these from the spec.

Comment (andreancardona) [inline .github/workflows/RFC-2119-usage.yml:16]:
```suggestion
        diffDoesNotContain: " must | must not | shall | shall not | should | should not | may"
```

Comment (andreancardona) [reply .github/workflows/RFC-2119-usage.yml]: ✅  here is the issue to track this: https://github.com/w3c/aria/issues/1916

File: .github/workflows/RFC-2119-usage.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: "Check if PR introduces errant RFC-2119"
+on: 
+  pull_request:
+      types: [opened, edited, assigned, closed, review_requested, ready_for_review]
```
Comment (spectranaut) [inline .github/workflows/RFC-2119-usage.yml:4]: I'm a little confused about this list, I'm not super familiar with github workflows, but does this check need to be run in all these scenarios? shouldn't it only run if the PR is updated/edited? Or am I'm miss understanding this list..

**Review by spectranaut — CHANGES_REQUESTED** [review]
This is exciting! but I have one question.

File: .github/workflows/RFC-2119-usage.yml (hunk 2 above)
Comment (ariellalgilmore) [reply .github/workflows/RFC-2119-usage.yml]: agree! i updated it just to include `opened/edited`

**Review by spectranaut — APPROVED** [review]
looks great!

**Review by pkra — APPROVED** [review]

**Pull request merged**